modules/core/src/com/haulmont/cuba/core/app/Emailer.java

```diff
@@ -553,14 +563,16 @@ protected void markAsNonSent(SendingMessage sendingMessage) {
         }
     }
 
-    protected SendingMessage convertToSendingMessage(String address, String from, String caption, String body,
+    protected SendingMessage convertToSendingMessage(String address, String from, String cc, String bcc, String caption, String body,
```

Comment:
Can we simplify this? The method already has so many parameters

Reply:
@jreznot It's not a public method, let's do it as a separate effort.